src/DataSource/Types.hs

```diff
@@ -172,6 +176,8 @@ data ZendeskAPIUrl
     deriving (Eq, Generic)
 
 showURL :: ZendeskAPIUrl -> Text
+showURL AgentGroupURL                       = "/groups/41693007/users.json" -- magic number..?
+-- users.json?role[]=admin&role[]=agent returns InvalidUrl
```

Comment:
I've tried using `users.json?role[]=admin&role[]=agent` to fetch the tickets it returns `Invalid Url` so I opted to use `/api/v2/users/{user_id}/groups.json` but in that case, we'll now have magic number.

Reply:
Hm. Well, you can use the `GroupId` and use a parameter, which would sort out the issue above and make testing easier, but don't include it in any configuration, send it as a magic number from some high-level module like `Lib`.

Reply:
Okay! I've kind of solved this.